flybywiresim/a32nx
# fix(apu): bleed available while in cooldown

## Summary of Changes
As the default APU would keep supplying bleed air while the APU BLEED pb was in the ON position, it was possible to start an engine during the cooldown phase. This should not be possible, as the APU doesn't provide BLEED while in cooldown.

This PR fixes the issue by triggering the opening and closing of the in-sim APU's BLEED valve based on the actual availability of our own APU.

## Additional context



Discord username (if different from GitHub): SjotgunSjonnie#9623

## Testing instructions
Steps to reproduce:

1. Start the APU.
2. Set APU BLEED to ON.
3. Turn the MASTER SW to OFF.

You end up in this state:
![image](https://user-images.githubusercontent.com/1074786/106443759-2d61b600-647d-11eb-911c-7005d8bcd945.png)


4. Try to start an engine.

Expected: The engine's N2 doesn't rise.
Actual: The engine's N2 rises.


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build script will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, click on the **Artifacts** drop down and click the **A32NX** link
